src/components/component/items/ItemCard.jsx

```
@@ -0,0 +1,56 @@
+              <source src={item.thumbnailVideo} type="video/mp4" alt={item.title} />
+            </video>
+          ) : (
+            <img src='/img/nothing.png' alt={item.title} />
+          )
+        }
+        <span>{item.title}</span>
+      </Link>
+      <div className='item-information'>
+        {item.teacher ? <p>{item.teacher}</p> : <p>&nbsp;</p>}
+        <span className='item-rating'>
+          <svg width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)